docs/background_jobs.md

```diff
@@ -0,0 +1,21 @@
+# Background jobs
+
+Sidekiq is the background job processor we use that handles long-running tasks, such as sending emails or processing images, asynchronously.
+
+`AttachProfilePictureWorker` is a Sidekiq worker that downloads an image from a given URL and attaches it to the corresponding user record.
```

Comment:
have a heading for jobs that we currently have in the platform already and this one being one  and what it does

Reply:
also mention about how the emails by default are sent in the background if deliver_later is used and how sidekiq handles that